### PR TITLE
Event Publisher for the tryit application

### PR DESCRIPTION
**Pull request by ashanthamara**
### Related Issues
- Issue [#11318](https://github.com/wso2-enterprise/asgardeo-product/issues/11318)

### CODE REVIEW

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 1]
```diff
@@ -129,7 +129,7 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         template,
         readOnly,
         urlSearchParams,
-        [ "data-testid" ]: testId
+        ["data-testid"]: testId
```
Comment (brionmario) [inline apps/console/src/features/applications/components/edit-application.tsx:132]: Add a config for your VSCode to not to autoformat `tsx|jsx|js|ts` files. Revert all other white space removals.

Comment (ashanthamara) [reply apps/console/src/features/applications/components/edit-application.tsx]: Fixed

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx [hunk 2]
```diff
@@ -59,205 +63,232 @@ interface SignInMethodLandingPropsInterface extends SBACInterface<FeatureConfigI
...
-                                        image={ getAuthenticatorIcons().basic }
+                                        image={getAuthenticatorIcons().basic}
                                         imageSize="mini"
-                                        header={ t(
+                                        header={t(
                                             "console:develop.features.applications.edit.sections.signOnMethod." +
-                                                "sections.landing.flowBuilder.types.defaultConfig.heading"
-                                        ) }
-                                        description={ t(
+                                            "sections.landing.flowBuilder.types.defaultConfig.heading"
```
Comment (DonOmalVindula) [inline apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx:109]: Any reason why the description was changed here?

**Review by DonOmalVindula — APPROVED** [review]

File: apps/console/src/features/applications/components/application-danger-zone.tsx [hunk 3]
```diff
@@ -89,6 +95,10 @@ export const ApplicationDangerZoneComponent: FunctionComponent<ApplicationDanger
  
                 setShowDeleteConfirmationModal(false);
                 onDelete();
+
+                eventPublisher.publish("application-delete", {
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/application-danger-zone.tsx:99]: eventPublisher.publish(
          "application-delete",
          { "client-id": clientId }
);

**Pull request merged**